main.go

```diff
@@ -75,8 +75,10 @@ func getConfig() (*config, error) {
 	}
 
 	// if no env. variable is set, this is the default repo watch list
+	// This list must be kept in sync with: release_tool.py --list git
```

Comment:
Hehehe. This I gave it a though, actually. What stopped me is that the configuration is taken once, while this (getting the repo list from the tool) should be done in the loop after every pull of integration repo (we pull all the time to have latest master).

I have it in my paper org as a possible value add follow-up... But I think we should merge this as is, so that we get the fix in place ASAP.